l10n_de_steuernummer/models/res_partner.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 Antonio Espinosa <antonio.espinosa@tecnativa.com>
+# Copyright 2017 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import re
+from odoo import api, models
+
+try:
+    import vatnumber
+except ImportError:
+    vatnumber = None
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    @api.model
+    def check_vat_de(self, vat):
+        res = False
+        if getattr(vatnumber, 'check_vat_de', None):
+            res = vatnumber.check_vat_de(vat)
+        if not res:
+            # This vat is not a Steue-IdNr., so if it is 10 or 11 digit
```

Comment:
@cubells I think that's not correct. Following grammar rules(https://rayli.net/blog/life/when-to-use-a-or-an/#First_why_the_confusion) a word like Steuer doen't begin with a vocal sound (although we spanish speaker tend to pronounce such words more like 'es...', than the proper 'ss'.